src/NzbDrone.Update/UpdateEngine/InstallUpdateService.cs

```diff
@@ -106,7 +109,31 @@ public void Start(string installationFolder, int processId)
             }
             finally
             {
-                _startNzbDrone.Start(appType, installationFolder);
+                if (OsInfo.IsWindows)
+                {
+                    _startNzbDrone.Start(appType, installationFolder);
+                }
+                else
+                {
+                    _terminateNzbDrone.Terminate(processId);
+
+                    _logger.Info("Waiting for external auto-restart.");
+                    for (int i = 0; i < 5; i++)
+                    {
+                        System.Threading.Thread.Sleep(i * 1000);
```

Comment:
5 seconds did seem a bit low, but we'd only be waiting 10 seconds, since its wait 0, 1, 2, 3, 4 seconds. Maybe you meant `(i + 1) * 1000` ?